autofile/autofile_test.go

```diff
@@ -1,12 +1,14 @@
+// nolint: goimports
```

Comment:
why? we should replace the `.` with `cmn`

Reply:
this file is not imported anywhere, so goimports throws

Reply:
huh? it's a test, why would it be imported anywhere?

Reply:
it wouldn't but i guess goimports isn't that smart